### DISCUSSION TITLE
Japanese Translation Contribution

### DISCUSSION THREAD

**Discussion by orandasoft** [Koenkk/zigbee2mqtt#10748] [General]
I wanted to contribute on a Japanese translation, but there is no option to contribute to a Japanese translation. I contacted POEditor and they told be that if I like to participate in the translation of a project hosted on their platform, I need to be invited by the owner or admin of the project.

**Comment by sjorge** [comment]
You probably want to report that here: https://github.com/nurikk/zigbee2mqtt-frontend/issues

The frontend is a seperate project.
